src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -1,13 +1,23 @@
 from __future__ import absolute_import
 import six
 
+from django.db import transaction, IntegrityError
 from django.db.models import Q
+from rest_framework import serializers
+from rest_framework.response import Response
+from django.conf import settings
 
-from sentry.api.bases.organization import (OrganizationEndpoint, OrganizationPermission)
+
+from sentry import roles
+from sentry.api.bases.organization import (
+    OrganizationEndpoint, OrganizationPermission)
 from sentry.api.paginator import OffsetPaginator
 from sentry.api.serializers import serialize
-from sentry.models import OrganizationMember
+from sentry.api.serializers.rest_framework import ListField
+from sentry.models import AuditLogEntryEvent, OrganizationMember, OrganizationMemberTeam, Team, TeamStatus
 from sentry.search.utils import tokenize_query
+from sentry.signals import member_invited
+from .organization_member_details import get_allowed_roles
```

Comment:
I don't really like importing this out of another view. I won't block it, but this is a code smell.

Reply:
true :/

Reply:
better to have it in a shared utility file, or what?

Reply:
Or a method on a base class. But it seems like it'd have a place in some util that already exists. Or a method on `Organization`.